cli/dts/lib.deno.ns.d.ts

```diff
@@ -86,93 +112,225 @@ declare interface PerformanceMeasureOptions {
+    /**
+     * Raised when the underlying operating system reports an `ENOTCONN` error.
+     *
+     * @category Errors */
     export class NotConnected extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when attempting to open a server listener on an address and port
+     * that already has a listener.
+     *
+     * @category Errors */
     export class AddrInUse extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when the underlying operating system reports an `EADDRNOTAVAIL`
+     * error.
+     *
+     * @category Errors */
     export class AddrNotAvailable extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when trying to write to a resource and a broken pipe error occurs.
+     * This can happen when trying to write directly to `stdout` or `stderr`
+     * and the operating system is unable to pipe the output for a reason
+     * external to the Deno runtime.
+     *
+     * @category Errors */
     export class BrokenPipe extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when trying to create a resource, like a file, that already
+     * exits.
+     *
+     * @category Errors */
     export class AlreadyExists extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when trying to internally convert data from an resource into a
+     * string. Deno expects source data to be encoded as UTF-8 and will error
+     * if the data is not valid UTF-8.
+     *
+     * @category Errors */
     export class InvalidData extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when the underlying operating system reports that an I/O operation
+     * has timed out (`ETIMEDOUT`).
+     *
+     * @category Errors */
     export class TimedOut extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when the underlying operating system reports an `EINTR` error. In
+     * many cases, this underlying IO error will be handled internally within
+     * Deno, or result in an @{link BadResource} error instead.
+     *
+     * @category Errors */
     export class Interrupted extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when expecting to write to a IO buffer resulted in zero bytes
+     * being written.
+     *
+     * @category Errors */
     export class WriteZero extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when attempting to read bytes from a resource, but the EOF was
+     * unexpectedly encountered.
+     *
+     * @category Errors */
     export class UnexpectedEof extends Error {}
-    /** @category Errors */
+    /**
+     * The underlying IO resource is invalid or closed, and so the operation
+     * could not be performed.
+     *
+     * @category Errors */
     export class BadResource extends Error {}
-    /** @category Errors */
+    /**
+     * Raised in situations where when attempting to load a dynamic import,
+     * too many redirects were encountered.
```

Comment:
All hyper errors are coerced into this type, so `Deno.serveHttp` can also return it. Unrelated, but I feel like dynamic imports should not be returning this error (and use `TypeError` instead).